### PR TITLE
add default_values

### PR DESCRIPTION
**Pull request by dustjr1565**
1. add default_values
```
hub = UltralyticsHub.new(...)
hub.default_values # output: {'epochs': 50, 'image_size': [640, 640], ...}
```

2. add config.py about each model

### CODE REVIEW

File: waffle_hub/hub/adapter/tx_model/tx_model_hub.py [hunk 1]
```diff
@@ -204,6 +204,10 @@ def get_metrics(self) -> list[dict]:
 
         return metrics
 
+    @property
+    def default_values(self):
```
Comment (ZeroAct) [inline waffle_hub/hub/adapter/tx_model/tx_model_hub.py:208]: `default_params` would be better choice 😊

File: waffle_hub/hub/adapter/hugging_face/config.py
Comment (ZeroAct) [inline waffle_hub/hub/adapter/hugging_face/config.py:1]: Use `TrainConfig` instead of dictionary.

**Review by ZeroAct — CHANGES_REQUESTED** [review]

File: waffle_hub/hub/adapter/tx_model/tx_model_hub.py (hunk 1 above)
Comment (dustjr1565) [reply waffle_hub/hub/adapter/tx_model/tx_model_hub.py]: done

File: waffle_hub/hub/adapter/hugging_face/config.py
Comment (dustjr1565) [reply waffle_hub/hub/adapter/hugging_face/config.py:1]: done

**Comment by ZeroAct** [comment]
@dustjr1565 
It seems some codes are not latest version.
please merge latest branch

**Review by ZeroAct — APPROVED** [review]

**Pull request merged**